docs/reference/index.md

```diff
@@ -11,19 +11,20 @@ parent = "smn_compose_ref"
 
 ## Compose CLI reference
 
-The following pages describe the usage information for the [docker-compose](/reference/docker-compose.md) subcommands. You can also see this information by running `docker-compose [SUBCOMMAND] --help` from the command line.
+The following pages describe the usage information for the [docker-compose](reference/docker-compose.md) subcommands. You can also see this information by running `docker-compose [SUBCOMMAND] --help` from the command line.
+
+* [build](../reference/build.md)
+* [help](../reference/help.md)
+* [kill](../reference/kill.md)          
+* [ps](../reference/ps.md)
+* [restart](../reference/restart.md)
+* [run](../referencerun.md)
```

Comment:
Thanks...I was thinking last night Friday late is a bad time to do this....I need a reviewer.  Viola!